packages/ui-view/src/View/index.js

```diff
@@ -64,29 +64,36 @@ category: components
 class View extends Component {
```

Comment:
This is just a huge auto-formatting change?

Reply:
Answered in https://github.com/instructure/instructure-ui/pull/157#discussion_r527562594.